package.json

```diff
@@ -38,7 +38,7 @@
     "@formatjs/intl-numberformat": "^6.2.5",
     "@formatjs/intl-pluralrules": "^4.0.13",
     "@onfido/react-native-sdk": "^1.3.3",
-    "@react-native-community/async-storage": "^1.11.0",
+    "@react-native-async-storage/async-storage": "^1.15.5",
```

Comment:
Hm, so why are we installing this lib directly? I'm not sure why the previous lib was referenced here either. AFAIK, the only way we should be interfacing with AsyncStorage is through Onyx.

Reply:
I'm not seeing it directly imported into any files, so can this just be removed?

Reply:
This is related to the comment I've made on peerDependencies on the other PR
We need to have it listed as a project dependency

We also have one place where it's imported - inside __mocks__ 
The e2e tests that use Onyx stub AsyncStorage from E.cash __mocks__